doc/proposals/README.md

```diff
@@ -0,0 +1,5 @@
+# Proposals
+
+This directory contains design documents for accepted proposals. To propose a new feature use the [template document][template]. Note that if the proposal gets accepted, the proposal implementation owner is expected to carry out the implementation.
```

Comment:
I think we should add that someone can create a proposal and not implement it if someone is willing to take it, but this field must be filled in before merging?